packages/block-editor/src/components/rich-text/toolbar-button.js

```diff
@@ -17,7 +17,7 @@ export function RichTextToolbarButton( { name, shortcutType, shortcutCharacter,
 	}
 
 	return (
-		<Fill name={ fillName }>
+		<Fill name={ fillName } key={ fillName }>
```

Comment:
I don't think it is going to work as expected, it doesn't change anything if there is only one item rendered.

Reply:
Hi @gziolo, without this change sometimes some strange behaviors happen. E.g: the color indicator of other blocks don't disappear:
![Jan-20-2020 14-50-51](https://user-images.githubusercontent.com/11271197/72736221-3c890500-3b95-11ea-9a39-b3bbedb74ab4.gif)


With this change, things work as expected.

Reply:
This is probably caused by https://github.com/WordPress/gutenberg/pull/16014#discussion_r374876562.

The `Fill` doesn't re-render after a `name` change to `.undefined,` and we accumulate format edit components from multiple selections.

If we follow the pattern of other formats of not rendering the button at all, then this key won't be needed.

Reply:
Hi @epiqueras, we need to render the button even if it is not active (the format is not enabled) so the user can press the button to enable the format. Other formats also render the button even if not active.

Reply:
They either always render it or render different ones depending on some conditions, but they never change the name dynamically.